README.ru.md

```diff
@@ -71,20 +75,235 @@ optimizt `find . -type f -name '*.jpg'`
+2. С помощью _Command Palette_ выберите пункт `>Tasks: Run Task`.
+3. Выберите нужную задачу.
+
+#### Горячие клавиши
+
+Добавить горячую клавишу для задачи можно выбрав пункт `>Preferences: Open Keyboard Shortcuts (JSON)` в _Command Palette_.
+
+Пример добавления горячей клавиши для запуска задачи «optimizt: Optimize Image (lossless)»:
+
+```javascript
+// Place your key bindings in this file to override the defaults
+[
+  {
+    "key": "ctrl+l",
+    "command": "workbench.action.tasks.runTask",
+    "args": "optimizt: Optimize Image (lossless)"
+  }
+]
+```
+
+</details>
+
+### Плагин для Sublime Text 3
+
+<details>
+
+<summary>Детали</summary>
+
+Пути расположения пользовательских настроек:
+
+- macOS: `~/Library/Application Support/Sublime Text 3/Packages/User`
+- Linux: `~/.config/sublime-text-3/Packages/User`
+- Windows: `%APPDATA%\Sublime Text 3\Packages\User`
+
+#### Добавьте плагин
+
+В директории настроек создайте файл `optimizt.py` со следующим содержимым:
+
+```python
+import os
+import sublime
+import sublime_plugin
+
+optimizt = "~/.nodenv/shims/optimizt"
+
+class OptimiztCommand(sublime_plugin.WindowCommand):
+  def run(self, paths=[], options=""):
+    if len(paths) < 1:
+      return
+
+    safe_paths = ["\"" + i + "\"" for i in paths]
+    shell_cmd = optimizt + " " + options + " " + " ".join(safe_paths)
+    cwd = os.path.dirname(paths[0])
+
+    self.window.run_command("exec", {
+      "shell_cmd": shell_cmd,
+      "working_dir": cwd
+    })
+```
+
+В переменной `optimizt` пропишите путь до исполняемого файла, который можно получить с помощью выполнения команды `command -v optimizt` в терминале.
+
+#### Интегрируйте плагин в контекстное меню сайдбара
+
+В директории настроек создайте файл `Side Bar.sublime-menu` со следующим содержимым:
+
+```json
+[
+    {
+        "caption": "Optimizt",
+        "children": [
+          {
+              "caption": "Optimize Images",
+              "command": "optimizt",
+              "args": {
+                "paths": [],
+                "options": "--verbose"
+              }
+          },
+          {
+              "caption": "Optimize Images (lossless)",
+              "command": "optimizt",
+              "args": {
+                "paths": [],
+                "options": "--lossless --verbose"
+              }
+          },
+          {
+              "caption": "Create WebP",
+              "command": "optimizt",
+              "args": {
+                "paths": [],
+                "options": "--webp --verbose"
+              }
+          },
+          {
+              "caption": "Create WebP (lossless)",
+              "command": "optimizt",
+              "args": {
+                "paths": [],
+                "options": "--webp --lossless --verbose"
+              }
+          }
+        ]
+    }
+]
+```
+
+#### Как использовать
+
+Вызовите контекстное меню на файле или директории и запустите необходимое:
+
+<img src="images/st_sidebar_menu.png" width="55%">
```

Comment:
I guess you can just cut the middle of the image instead of blurring it.

Reply:
Done!